apps/ui/src/Pages/SignIn/SignIn.tsx

```diff
@@ -33,23 +28,27 @@ const SignIn = ({ setMenuVisible }: SignInProps) => {
   const [isSubmitEnabled, setIsSubmitEnabled] = useState(false);
 
   useEffect(() => {
```

Comment:
You don't need this useEffect, you can check for errors in the onChange function